scripts/skimNtuple_mib.py

```diff
@@ -37,8 +37,6 @@ def parseInputFileList (fileName) :
     usage = 'usage: %prog [options]'
     parser = optparse.OptionParser(usage)
     parser.add_option ('-i', '--input'     , dest='input'     , help='input folder'                          , default='none')
-    parser.add_option ('-Y', '--year'      , dest='year'      , help='year'                                  , default='2018')
-    parser.add_option ('-A', '--APV'       , dest='isAPV'     , help='isAPV'                                 , default=False )
```

Comment:
Instead of modifying ```scripts/skimNtuple_mib.py```, why not using ```scripts/skimNtuple.py```, which is fully up-to-date, and avoiding storing many very similar scripts (error prone)?